### DISCUSSION TITLE
Twinkle Tray resets back to previous brightness right after I change it to a new one

### DISCUSSION THREAD

**Discussion by Dimitrije25** [xanderfrangos/twinkle-tray#719] [Q&A]
When I change my brightness from example from 50, back to 20, my brightness does change to 20, but then immediately goes back to 50, and it happens like that 2-3 times until I change it back to 20 for the 3-4th time. 

This is present constantly, are there any solutions regarding this?

**Comment by pureby** [comment]
1. Restart your PC, see if that fixes the problem.
2. Experiment with different values for _Settings > Monitor Settings > Brightness update rate_
3. Try enabling _Settings > DDC/CI Features > Get Current Brightness_
4. Do you have any other software installed that can control brightness? Disable it.
5. If you're using a laptop, it is possible that you have drivers installed that override brightness. Try disabling "automatic brightness" setting on your laptop and closing all helper-applications provided by your laptop manufacturer (including ones running in the background, using Task Manager).

**Reply by Dimitrije25** [reply]
Thank you! 

1. This issue has been happening for months, so I am not sure this would solve it. 
2. I did change the update rate, but will try to adjust it. 
3. Will try that, thanks 
4. I do not have anything else installed

**Reply by pureby** [reply]
@Dimitrije25 You're welcome! Is your problem solved? If yes, please report what solved your problem, it might might help other people in the future ;-)

**Reply by Dimitrije25** [reply]
I will see for the next days, and will close as successful if no further issues, thanks.

**Reply by Dimitrije25** [reply]
I can say that testing it after a few days, the issue still remains but its basically smaller. 
The issue is, changing the brightness too fast, so I had to drop it down to ''normal speed'', over which the issue is still present, but in less quantity.

**Comment by cars11** [comment]
yeah i have this same problem. if change it too fast it and close the tray window right away it does nothing at all. 
i have to click on twinkletray icon a second time, then it applies the value i set the first time. This is especially annoying while playing a video game, as I often like to do quick adjustments throughout a session and I basically never can.

seems the program needs like a full 3 seconds of being open after you change a value.
disappointing :(
